cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/store/AppMetadataStore.java

```diff
@@ -1750,6 +1764,26 @@ public void scanActiveRuns(ProgramId programId, Consumer<RunRecordDetail> consum
     }
   }
 
+  /**
+   * Scans active runs of the given program irrespective of the versions of the programs.
+   * Active runs means program run with status STARTING, PENDING, RUNNING or SUSPENDED.
+   * This method is similar to the {@link #scanActiveRuns(ProgramId, Consumer)},
+   * but consuming results in a streaming fashion
```

Comment:
I think scanActiveRuns also does it in a streaming fashion

Reply:
Updated the javadoc